README.md

```diff
@@ -34,10 +34,11 @@ imagelab.find_issues(issue_types)
 imagelab.report(issue_types.keys())
 ```
 
-More on how to get started with CleanVision: 
+More on how to get started with CleanVision:
 
-- [example script](https://github.com/cleanlab/cleanvision/blob/main/examples/run.py)
-- [jupyter notebook demo](https://github.com/cleanlab/cleanvision/blob/main/examples/demo.ipynb)
+- [Example script](https://github.com/cleanlab/cleanvision/blob/main/examples/run.py)
+- [Jupyter notebook tutorial](https://github.com/cleanlab/cleanvision/blob/main/examples/demo.ipynb)
+- [Documentation](https://cleanvision.readthedocs.io/en/latest/)
```

Comment:
```suggestion
- [Documentation](https://cleanvision.readthedocs.io/)
```

Reply:
seems like this shorter one should be the official link?

Reply:
shorter one redirects to latest, I think either of them works

Reply:
I am saying we should have one official link we consistently share everywhere, not multiple, and it should be the most concise one